code/game/objects/items/weapons/charter.dm

```diff
@@ -51,11 +51,6 @@
 	log_game("[key_name(user)] has proposed to name the station as \
 		[new_name]")
 
-	if(standard_station_regex.Find(new_name))
```

Comment:
Is standard_station_regex only used here? If so, you could remove it and any related code.